docs_src/microservices/device/details/ServiceMetrics.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Device Service - Service Metrics
+---
+
+# Device Service - Service Metrics
+
+All device services have the following built-in metrics:
+
+| Metric Name                   | Type      | Description                                                  |
+| ----------------------------- | --------- | ------------------------------------------------------------ |
+| EventsSent                    | bool      | Enable/disable reporting of the built-in **EventsSent** metric |
+| ReadingsSent                  | bool      | Enable/disable reporting of the built-in **ReadingsSent** metric |
+| LastConnected                 | bool      | Enable/disable reporting of the built-in **LastConnected** metric|
+| <CustomMetric>                | bool      | Enable/disable reporting of custom device service's custom metric. See Custom Device Service Metrics for more details. |
```

Comment:
This table is just the one already provided.
```suggestion
```

Reply:
removed